projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/enums/query-item-response-status.enum.ts

```diff
@@ -1,5 +1,6 @@
 export enum QueryItemResponseStatus {
   NEW = 'New',
   RESPONDED = 'Responded',
-  AWAITING = 'Awaiting Response'
+  AWAITING = 'Awaiting Response',
```

Comment:
Very minor, but the UX rule is sentence case everywhere, so this should be "Awaiting response"